dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -237,6 +237,14 @@ def set_code_review_groups
     render json: {result: 'invalid groups'}, status: 400
   end
 
+  # POST /api/v1/sections/<id>/code_review_enabled
+  def set_code_review_enabled
```

Comment:
I think we'll probably need some equivalent GET route as well, e.g. get_code_review_enabled, would that be part of a future change?

Reply:
you'll be able to get it by getting a section, I think that'll be sufficient since we get the section already? But we can add a separate route if we need to. I think @bencodeorg had thoughts on this.

Reply:
Oh gotcha, yes that works too. I guess a dedicated route could be useful if we have any additional logic here but if it's just reading from the section properties then getting reading it from the section data sounds fine to me.

Reply:
Yeah, I was imagining we just include it in the initial set of section data loaded into redux on page load when loading the teacher dashboard. I believe there was a requirement that we add a notification to the teacher dashboard that your code review groups will expire soon, so will need this information outside of when you have the code review groups modal open.